geomesa-fs/geomesa-fs-storage/geomesa-fs-storage-common/src/main/java/org/locationtech/geomesa/fs/storage/api/PartitionSchemeUtils.java

```diff
@@ -0,0 +1,53 @@
+/***********************************************************************
+ * Copyright (c) 2013-2018 Commonwealth Computer Research, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License, Version 2.0
+ * which accompanies this distribution and is available at
+ * http://www.opensource.org/licenses/apache2.0.php.
+ ***********************************************************************/
+
+package org.locationtech.geomesa.fs.storage.api;
```

Comment:
can we move this class to `org.locationtech.geomesa.fs.storage.common.interop` ? to keep packages separate per module and keep with our interop package naming conventions

Reply:
Renamed